AustrianPatient/AustrianPatient.xml

```diff
@@ -24,6 +24,7055 @@
   <type value="Patient" />
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Patient" />
   <derivation value="constraint" />
+  <snapshot>

```

Comment:
This is because of the tools we are currently using - both Forge and the IG publisher need the snapshot to properly compute the effective profile structure.